apps/teams-test-app/src/App.tsx

```diff
@@ -42,6 +43,35 @@ if (
   }
 }
 
+function handleMessageFromMockedHost(msg: MessageEvent): void {
+  if (!msg || !msg.data) {
+    console.warn('Unrecognized message format received by app, message being ignored. Message: %o', msg);
+    return;
+  }
+  console.log(`Received message from test host: ${msg.data}`);
+  // Ensure the message is from a trusted origin
```

Comment:
Is this comment accurate? I'm not sure I see how the code below this is ensuring the message is coming from a trusted origin.

Reply:
Maybe something like

```suggestion
  // Handle messages that are correctly formatted and for func values we recognize
```

Reply:
I think that comment is for user to make sure the message is from a trusted origin rather than saying the functionality of that piece of code lol.

Thank you @TrevorJoelHarris , I have taken your suggestion.